rosdep/python.yaml

```diff
@@ -1563,6 +1563,14 @@ python-easygui:
   debian: [python-easygui]
   fedora: [python-easygui]
   ubuntu: [python-easygui]
+python-elasticsearch:
+  debian:
+    buster: [python-elasticsearch]
+    stretch: [python-elasticsearch]
+  fedora: [python-elasticsearch]
```

Comment:
Fedora is now aliasing unversioned python package names to Python 3. There is no Python 2 package for Fedora.
```suggestion
```